wrapper-classes/BaseElement.js

```diff
@@ -0,0 +1,119 @@
+import { Timeouts } from '../constants/timeouts.js'
+
+class BaseElement {
+  constructor(name, selector) {
+    this.name = name
+    this.selector = selector
+  }
+
+  async getElement() {
+    return await $(this.selector)
+  }
+
+  async waitForExist() {
```

Comment:
it would be better to add these parameters: { timeout, reverse, interval } with some default values

Reply:
same is applicable to other similar methods